test/apiv2/12-imagesMore.at

```diff
@@ -86,6 +86,33 @@ t POST "libpod/images/scp/$IMAGE?destination=QA::" 500 \
 t DELETE libpod/images/$IMAGE 200 \
   .ExitCode=0
 
+# list images through the libpod endpoint should not return
+# IsManifestList (bool), Arch (string), and Os (string)
+podman pull -q $IMAGE
```

Comment:
why are we pulling a image here? don't we have already pulled on already, can you just move this before the API call to remove the image, every new pull makes the test slower and more flaky